_data/events_gdcr2018/London-GDS.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "GDCR18 @GDS London",
+  "url": "https://www.eventbrite.com/e/gdcr18-gds-london-tickets-52153436332",
```

Comment:
Doesn't exist for me

Reply:
Yes, the link to the event is not public yet. Should I get back to raise it again once the link goes public?
Thanks :)

Reply:
Yeah let's wait until people can actually sign up :+1:

Reply:
Eventbrite link has been updated :)